docs/modules/ROOT/pages/authorization.adoc

```diff
@@ -0,0 +1,90 @@
+= Authorization Methods to Access Cloud Storage
+
```

Comment:
You can use either a secret or `Service Accounts` to access your cloud storage, as detailed below.